getting-started/FAQ.md

```diff
@@ -27,7 +27,7 @@ Native k8s supports specifying a security context for the pod or container. It r
 * All the OS distributions do not support the LSMs consistently. For e.g, [GKE COS](https://cloud.google.com/container-optimized-os/) supports AppArmor while [Bottlerocket](https://aws.amazon.com/bottlerocket/) supports SELinux and BPF-LSM.
 * The Pod Security Context expect the security profile to be specified in its native language, for instance, AppArmor profile for AppArmor. SELinux profile if SELinux is to be used. The profile language is extremely complex and this complexity could backfire i.e, it could lead to security holes.
 * Security Profile updates are manual and difficult: When an app is updated, the security posture might change and it becomes difficult to manually update the native rules.
-* No alerting of LSM violation on managed cloud platforms: By default LSMs send logs to kernel auditd, which is not available on most managed cloud platforms.
+* No alerting of LSM violation on managed cloud platforms: By default LSMs send logs to kernel audit, which is not available on most managed cloud platforms.
```

Comment:
It's not a spelling mistake but an intended word. It means kernel audit daemon.

More about auditd at https://man7.org/linux/man-pages/man8/auditd.8.html

Reply:
Oh! I misinterpreted it, sorry